addons/dictionary_inspector/inspector_plugin.gd

```diff
@@ -24,8 +24,10 @@ func _can_handle(object):
 
 
 func _parse_property(object, type, path, hint, hint_text, usage, wide) -> bool:
-	if !typeof(object[path]) in supported_types:
-		return false
+	if object != null:
```

Comment:
Indented every single line right there 🤔

How about `if object == null: return false` and keep everything else

Reply:
Ok that sounds good, do you want me to make and update or just close this PR.

Reply:
As far as when it happens, is on ,my project where I have multiple objects.

Reply:
It's not too bad.